### PR TITLE
[docs] Replaced daemon flag from docker run command with background.

### PR DESCRIPTION
**Pull request by aishwarya24**
The yugabyted flag --daemon is deprecated - https://docs.yugabyte.com/preview/reference/configuration/yugabyted/#deprecated-flags

### CODE REVIEW

**Review by FranckPachot — COMMENTED** [review]
I don't think it should be removed but replaced by `--background=true`
The reason of deprecation is that it has been renamed

**Comment by aishwarya24** [comment]
> I don't think it should be removed but replaced by `--background=true` The reason of deprecation is that it has been renamed

Thanks @FranckPachot - will update it.

**Comment by ddhodge** [comment]
> I don't think it should be removed but replaced by `--background=true` The reason of deprecation is that it has been renamed

You mean false? --daemon was set to false. Like daemon, the default is true, and the point seemed to be to stop it running as a background process in Docker @FranckPachot

**Comment by FranckPachot** [comment]
Yes, you're right. Must be false to stay in foreground 🤦

File: docs/content/preview/quick-start/docker.md [hunk 1]
```diff
@@ -88,8 +88,7 @@ To create a 1-node cluster with a replication factor (RF) of 1, run the followin
 
 ```sh
 docker run -d --name yugabyte -p7000:7000 -p9000:9000 -p15433:15433 -p5433:5433 -p9042:9042 \
- yugabytedb/yugabyte:{{< yb-version version="preview" format="build">}} bin/yugabyted start \
- --daemon=false
+ yugabytedb/yugabyte:{{< yb-version version="preview" format="build">}} bin/yugabyted start \ --background=true
```
Comment (ddhodge) [inline docs/content/preview/quick-start/docker.md:91]:
```suggestion
 yugabytedb/yugabyte:{{< yb-version version="preview" format="build">}} bin/yugabyted start \ --background=false
```

**Review by ddhodge — APPROVED** [review]
LGTM

File: docs/content/preview/quick-start/docker.md [hunk 2]
```diff
@@ -88,8 +88,7 @@ To create a 1-node cluster with a replication factor (RF) of 1, run the followin
 
 ```sh
 docker run -d --name yugabyte -p7000:7000 -p9000:9000 -p15433:15433 -p5433:5433 -p9042:9042 \
- yugabytedb/yugabyte:{{< yb-version version="preview" format="build">}} bin/yugabyted start \
- --daemon=false
+ yugabytedb/yugabyte:{{< yb-version version="preview" format="build">}} bin/yugabyted start \ --background=false
```
Comment (FranckPachot) [inline docs/content/preview/quick-start/docker.md:91]: I seems that a new line is missing. Should be:
```
bin/yugabyted start 
\ --background=false
```

Comment (aishwarya24) [reply docs/content/preview/quick-start/docker.md:91]: Thanks @FranckPachot - addressed this in another [PR](https://github.com/yugabyte/yugabyte-db/pull/20109).

**Pull request merged**